storage/tests/unit/test_bucket.py

```diff
@@ -663,6 +666,27 @@ def test_cors_setter(self):
         self.assertEqual(bucket.cors, [CORS_ENTRY])
         self.assertTrue('cors' in bucket._changes)
 
+    def test_labels_getter(self):
+        NAME = 'name'
+        LABELS = {'color': 'red', 'flavor': 'cherry'}
+        properties = {'labels': LABELS}
+        bucket = self._make_one(name=NAME, properties=properties)
+        labels = bucket.labels
+        self.assertEqual(labels, LABELS)
+        # Make sure it was a copy, not the same object.
+        self.assertIsNot(labels, LABELS)
+
+    def test_labels_setter(self):
+        NAME = 'name'
+        LABELS = {'color': 'red', 'flavor': 'cherry'}
+        bucket = self._make_one(name=NAME)
+
+        self.assertEqual(bucket.labels, {})
+        bucket.labels = LABELS
+        self.assertEqual(bucket.labels, LABELS)
+        self.assertIsNot(bucket.labels, LABELS)
```

Comment:
@tseaver I really meant

```python
self.assertIsNot(bucket._properties['labels'], LABELS)
```

In it's current form, the getter indirection doesn't test the same concept